src/software/pipeline/main_cameraInit.cpp

```diff
@@ -471,6 +512,38 @@ int aliceVision_main(int argc, char **argv)
     const double diag24x36 = std::sqrt(36.0 * 36.0 + 24.0 * 24.0);
     camera::EIntrinsicInitMode intrinsicInitMode = camera::EIntrinsicInitMode::UNKNOWN;
 
+
+    std::unique_ptr<oiio::ImageInput> in(oiio::ImageInput::open(view.getImagePath()));
+
+    std::string imgFormat = in->format_name();
+
+    // check if a color profile is available for the image
+    if (!colorProfileDatabaseDirPath.empty() && (imgFormat.compare("raw") == 0))
+    {
+        bool colorProfileFound = false;
+        if (hasCameraMetadata)
+        {
+            std::pair<std::string, std::string> pairMakeModel = std::pair<std::string, std::string>(make, model);
+            std::list<std::string>::iterator it = findColorProfile(pairMakeModel, colorProfileList);
+
+            colorProfileFound = (it != colorProfileList.end());
+
+            if (colorProfileFound)
+            {
+                view.addMetadata("AliceVision:colorProfileFileName", *it);
+            }
+        }
+
+        #pragma omp critical
```

Comment:
#pragma omp atomic update

Reply:
omp atomic cannot be used with boolean. Change boolean in int to use atomic or keep critical.